pypasta/Repository/Mbox.py

```diff
@@ -16,10 +16,14 @@
 import os
 import pygit2
 import re
+import requests
+import hashlib
 
 from email.charset import CHARSETS
 from logging import getLogger
 from subprocess import call
+from urllib.parse import urljoin
+from datetime import datetime
```

Comment:
Hmm, above, we already import datetime. Importing it this way requires to maintain existing code.